Add Marching Orders, Clappy Trio and Rhythm Test stuff

GuardedLolz opened the pull request:
### Prerequisites
* [X] I have read and understood the [contribution guidelines](CONTRIBUTING.md)
* [X] This pull request follows the code style of the project
* [X] I have tested this feature thoroughly
  * VIDEO COMING SOON!

**Issues Fixed:** [Any issues fixed by this pull request]

### Changes Proposed in this Pull Request
* Add 3 new games - Marching Orders, GBA Clappy Trio (Fever sounds moved to the Fever folder), and Rhythm Test.

GuardedLolz committed:
Add Marching Orders, Clappy Trio and Rhythm Test stuff

chrislo27 reviewed (commented):
Looks good from a quick read through - could you find a way to make the two Clappy Trio icons differ obviously?

chrislo27 commented on core/assets/sounds/cues/marchingOrdersEn/data.json:
```diff
@@ -0,0 +1,212 @@
+{
+  "gameID": "marchingOrdersEng",
```
Please refer to the [language guidelines](https://github.com/chrislo27/RhythmHeavenRemixEditor2/wiki/Guidelines-and-Syntactical-Sugar#language-guidelines). Use `marchingOrdersEn` (you did this in data.json which is right but not in here for some reason).

GuardedLolz committed:
Fixed this cue

GuardedLolz committed:
Hey, those signs should be good for something!

GuardedLolz commented:
Done.

chrislo27 commented on core/assets/sounds/cues/marchingOrdersEn/data.json:
```diff
@@ -0,0 +1,212 @@
+  "series" : "tengoku",
+  "cues": [
+    {
+      "id": "marchingOrdersEng/attention1",
```
IDs are still on old Eng.

chrislo27 commented:
Also, the FEVER and GBA signs on Clappy Trio are impossible to read at a glance. It should be much more obvious.

GuardedLolz committed:
Fix the Marching Orders English database...

... but I just don't know what to do with the Clappy Trio icons... I
don't want to create Slenderlion.

chrislo27 commented:
I merged it because I am going to fix the errors by a certain time. There are several (missing Engs left over, icons not changed) so please next time fix all the errors.
